pkg/controller/jobs/rayjob/rayjob_controller.go

```diff
@@ -87,7 +88,7 @@ func (j *RayJob) IsSuspended() bool {
 }
 
 func (j *RayJob) IsActive() bool {
-	return j.Status.JobDeploymentStatus != rayv1.JobDeploymentStatusSuspended
+	return j.Status.JobDeploymentStatus != rayv1.JobDeploymentStatusNew
```

Comment:
Do we need this changes?

Reply:
well, that was the most important change actually
Because `JobDeploymentStatusSuspended` is not a initial status and the workload couldn't be created at all
Maybe it should be both like not new and not suspended, but definitively not the suspended only

Reply:
I am not even entirely sure if this should watch the JobDeploymentStatus, as the Job has its own Status.

Reply:
I think you right, it should be both.

https://github.com/kubernetes-sigs/kueue/blob/cea42f23253a4a3221ed529faef539cb61ec742f/pkg/controller/jobframework/interface.go#L57-L58

Reply:
cc: @andrewsykim